src/Chainweb/Miner/Core.hs

```diff
@@ -67,22 +66,14 @@ mine _ h = BA.withByteArray initialTargetBytes $ \trgPtr -> do
 
             -- inner mining loop
             --
-            -- We do 100000 hashes before we update the creation time.
-            --
-            let go 100000 !n = do
-                    -- update the block creation time
-                    ct <- getCurrentTimeIntegral
-                    injectTime ct buf
-                    go 0 n
```

Comment:
@larskuhtz and I discussed it, and decided that resetting the time-stamp "mid-mine" isn't useful.